applications_tests/lethe-fluid/gls_vof_1_isothermal_compressible_fluid.output

```diff
@@ -7,8 +7,8 @@ Running on 1 MPI rank(s)...
 ----------------------
 VOF Mass Conservation
 ----------------------
-   time    surface_fluid_0 mass_per_length_fluid_0 surface_fluid_1 mass_per_length_fluid_1 sharpening_threshold 
-0.0000e+00      6.5778e-01              7.8933e-01      3.4222e-01              3.4222e+02           5.0000e-01 
+   time    surface_fluid_0 mass_per_length_fluid_0 momentum-x_fluid_0 momentum-y_fluid_0 surface_fluid_1 mass_per_length_fluid_1 momentum-x_fluid_1 momentum-y_fluid_1 sharpening_threshold 
```

Comment:
Could it be useful to have to have the total mass and total momentum output as well?

Reply:
Someone can easily calculate those by adding the two columns in their own script. I'm not sure this is necessary :)